.travis.yml

```diff
@@ -36,6 +36,8 @@ env:
   - IMAGEMAGICK_VERSION=6.8.9-10 CONFIGURE_OPTIONS=--enable-hdri
   - IMAGEMAGICK_VERSION=6.9.10-61
   - IMAGEMAGICK_VERSION=6.9.10-61 CONFIGURE_OPTIONS=--enable-hdri
+  - IMAGEMAGICK_VERSION=7.0.8-61 CONFIGURE_OPTIONS=--disable-hdri
+  - IMAGEMAGICK_VERSION=7.0.8-61 CONFIGURE_OPTIONS=--enable-hdri
```

Comment:
`hdri` is enabled by default in IM7, right?

Reply:
Yup could be removed you prefer that.

Reply:
Yeah, I think it's fine to remove, unless there's a compelling reason to keep it. I could see maybe adding a comment to the end of the line if we want to clarify:

```yml
- IMAGEMAGICK_VERSION=7.0.8-61 CONFIGURE_OPTIONS # hdri is enabled by default on IM7
```

Reply:
We also don't have a comment for IM6 so better to only specify the non default value. Just pushed a commit to change this.